pylint/checkers/non_ascii_names.py

```diff
@@ -41,26 +41,6 @@ class NonAsciiNameChecker(base_checker.BaseChecker):
             NON_ASCII_HELP,
             {"old_names": [("C0144", "old-non-ascii-name")]},
         ),
-        # First %s will always be "file"
-        "W2402": (
-            (
-                '%s name "%s" contains a non-ASCII character. PEP 3131 only allows '
-                "non-ascii identifiers, not file names."
-            ),
-            "non-ascii-file-name",
-            (
-                # Some = PyCharm at the time of writing didn't display the non_ascii_name_loł
-                # files and had big troubles with git.
-                # Probably only a bug shows the problem quite good.
-                # That's also why this is a warning and not only a convention!
-                "Some editors don't support non-ASCII file names properly. "
-                "Even though Python supports UTF-8 files since Python 3.5 this isn't "
-                "recommended for interoperability. Further reading:\n"
-                "- https://peps.python.org/pep-0489/#export-hook-name\n"
-                "- https://peps.python.org/pep-0672/#confusing-features\n"
-                "- https://bugs.python.org/issue20485"
```

Comment:
Hmm, I did not realize this was well thought out. What happen if we name a file ``non_ascii_name_loł`` in the doc example ?

Reply:
The documentation could be changed so that there is a link to non-ascii characters so that a bad.py or good.py file is not needed.

Reply:
I also reverted the change that removed non-ascii-file-name (for now)